shared-helpers/src/regions.ts

```diff
@@ -14,7 +14,7 @@ export const regionImageUrls: Map<Region, string> = new Map([
   ],
   [
     Region.Southwest,
-    "https://www.theneighborhoods.org/sites/the-neighborhoods/files/2020-10/Southwest-Mural_1.jpg",
+    "http://www.theneighborhoods.org/sites/the-neighborhoods/files/2020-10/Southwest-Mural_1.jpg",
```

Comment:
this may cause our site to be considered unsecure since we'd be making an http call for an image

wondering if we can ask to move this image under cloudinary

Reply:
Can you provide some resources on the security concerns? Preferably https 🥲

Reply:
Unfortunately it looks like Yazeed might be on to something here. https://webmasters.stackexchange.com/questions/87987/https-connection-is-not-safe-due-to-images. Could we potentially reach out to someone to see if we can use this image and host it locally?